README.md

```diff
@@ -0,0 +1,4 @@
+# Tarantool Actions
+
+A repository with commonly used GitHub actions used in 
```

Comment:
`... used ... used ...` - probably we can fix it so:

A repository with commonly used GitHub actions for the `tarantool` organization on GitHub.